packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -40,6 +40,7 @@ library TaikoData {
         uint64 initialUncleDelay;
         bool enableTokenomics;
         bool enablePublicInputsCheck;
+        bool enableProofValidation;
```

Comment:
added this so I can skip actually making any merkle proofs, zk proofs, anchor transactions, and receipt generation in testing.